database/migrations/2022_08_15_071659_drop_column_name_in_users_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->dropColumn('name');
+            $table->string('first_name');
+            $table->string('last_name');
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->string('name');
+            $table->dropColumn(['first_name', 'last_name'],);
```

Comment:
```suggestion
            $table->dropColumn([
                'first_name',
                'last_name',
            ]);
```
fix tương tự các chỗ khác

Reply:
Done ạ